src/Utils/Hooks/useJump.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC, useEffect } from "react"
+import { themeProps } from "@artsy/palette-tokens"
+import { useNavBarHeight } from "Components/NavBar/useNavBarHeight"
+import { getOffsetTopForSticky, useSticky } from "Components/Sticky"
+import { __internal__useMatchMedia } from "./useMatchMedia"
+
+interface UseJump {
+  behavior?: ScrollBehavior
+  offset?: number
+}
+
+const NAMESPACE = "JUMP"
+
+/**
+ * Returns a `jumpTo` function that scrolls with an automatically calculated offset
+ * based on the nav and the current state of the sticky component.
+ */
+export const useJump = ({ behavior = "smooth", offset = 0 }: UseJump = {}) => {
```

Comment:
Thoughts on this living in Palette?

Reply:
It's dependent on the `Sticky` implementation.